### PR TITLE
Make it responsive

### PR DESCRIPTION
**Pull request by Mist12345**
I removed the grid as it didn't seem to make a difference. I changed the name of body to container as it was causing me an error. I also added media queries.

### CODE REVIEW

File: components/MainForm.js [hunk 1]
```diff
@@ -1,49 +1,80 @@
...
+  @media only screen and (min-width: 768px) {
+    font-size:1m;
+  }  
 `
 
 const CharacterInput = styled.input.attrs({
   // we can define static props
   type: 'password',
-  placeholder: 'Pasta your password here',
+  placeholder: 'Paste your password here',
```
Comment (spences10) [inline components/MainForm.js]: Pasta is intentional 😉

File: components/MainForm.js [hunk 2]
```diff
@@ -1,49 +1,80 @@
...
 
   // or we can define dynamic ones
   margin: props => props.size || '10px',
-  padding: props => props.size || '10px'
+  padding: props => props.size || '10px',
```
Comment (spences10) [inline components/MainForm.js]: I know there is Prettier config or guidelines as such but my preference is no comma dangle

File: components/MainForm.js [hunk 3]
```diff
@@ -1,49 +1,80 @@
...
 const CharacterSelect = styled.select`
-  font-size: 1.5em;
   border: 2px solid purple;
   border-radius: 5px;
   justify-self: start;
+  font-size: 1.5em;
+
+//for tablets
+/*  @media only screen and (max-width: 425px) and (min-width: 768px) {
+    font-size: 1.5em;
```
Comment (spences10) [inline components/MainForm.js]: Something a bit off here with the select box 🤔 in mobile view

image

**Comment by Mist12345** [comment]
@spences10 I've added some commits. Fixing the select options font size, however I was thinking it would be better if the select was changed to input type=number.

**Comment by Mist12345** [comment]
I feel that it's more intuitive and little bit faster. Honestly, now that I think about it the benefit isn't worth the work.

**Pull request merged**